.changeset/serious-pillows-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/identity-apps-core": patch
```

Comment:
Since this is a new feature, shall we bump to minor?

Reply:
Addressed